src/components/messages/message.tsx

```diff
@@ -32,11 +32,30 @@ export const Message: FC<Props> = ({ message, color, characterName }) => {
           <Typography variant="body1" textAlign={textAlign}>
             {message.message}
           </Typography>
-          <Typography variant="caption" textAlign={"right"}>
-            {timeStamp}
-          </Typography>
+          <Stack>
+            <Typography variant="caption" textAlign={"right"}>
+              {timeStamp}
+            </Typography>
+          </Stack>
         </Stack>
       </Stack>
     </Stack>
   );
 };
+
+const getCharacter = (characterName: string) => {
```

Comment:
```suggestion
const getCharacter = (characterName: CharacterName) => {
```
where `type CharacterName = "hal" | "ash" | ...`
defined that type in the types dir since we'll probably need it in more places

Reply:
Just saw that we actually have that type already so use it please